server/repository/migrations/postgres/2024-08-02T17-00_migration_enum_keys/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TYPE key_type ADD VALUE IF NOT EXISTS 'DATABASE_MIGRATED_FROM_PRE_RELEASE';
+
+ALTER TYPE key_type ADD VALUE IF NOT EXISTS 'DATABASE_MIGRATION_ERROR';
```

Comment:
I thought this was the cleanest way to get these key types in before the new migration process runs...